core/templates/dev/head/dashboard/DashboardServices.js

```diff
@@ -0,0 +1,76 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Standalone services for the creator dashboard page.
+ */
+
+oppia.constant('EXPLORATIONS_SORT_BY_KEYS', {
```

Comment:
I still lean towards having a constant. This is because, we don't want to have to call the function with "magic strings" at various places. However, we should implement the constant such that the strings are only accessed using this constant. (Let's discuss this further tomorrow during our meeting).